### PR TITLE
Add `ListAccessKeysBulk` API for builtin user access keys

### PR DESCRIPTION
**Pull request by taran-p**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Adds `ListAccessKeysBulk` API to list access keys for specified or all builtin users. Analogous to the `ListAccessKeysLDAPBulk` API but for builtin users.

## Motivation and Context
There is no no equivalent for `ListAccessKeysLDAPBulk` for builtin users. 

## How to test this PR?
Use minio/mc#5038 with `mc admin accesskey list`.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/admin-handlers-users.go [hunk 1]
```diff
@@ -1165,6 +1165,173 @@ func (a adminAPIHandlers) DeleteServiceAccount(w http.ResponseWriter, r *http.Re
...
+
+	cred, owner, s3Err := validateAdminSignature(ctx, r, "")
+	if s3Err != ErrNone {
+		writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(s3Err), r.URL)
+		return
+	}
+
+	userList := r.Form["users"]
+	isAll := r.Form.Get("all") == "true"
+	onlySelf := !isAll && len(userList) == 0
```
Comment (shtripat) [inline cmd/admin-handlers-users.go]: `selfOnly` sounds better

File: cmd/admin-handlers-users.go [hunk 2]
```diff
@@ -1165,6 +1165,173 @@ func (a adminAPIHandlers) DeleteServiceAccount(w http.ResponseWriter, r *http.Re
...
+	var checkedUserList []string
+	if isAll {
+		users, err := globalIAMSys.ListUsers(ctx)
+		if err != nil {
+			writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
+			return
+		}
+		for user := range users {
+			checkedUserList = append(checkedUserList, user)
+		}
```
Comment (shtripat) [inline cmd/admin-handlers-users.go:1246]:
```suggestion
		checkedUserList = append(checkedUserList, users...)
```

File: cmd/admin-handlers-users.go [hunk 3]
```diff
@@ -1165,6 +1165,173 @@ func (a adminAPIHandlers) DeleteServiceAccount(w http.ResponseWriter, r *http.Re
...
+		return
+	}
+
+	cred, owner, s3Err := validateAdminSignature(ctx, r, "")
+	if s3Err != ErrNone {
+		writeErrorResponseJSON(ctx, w, errorCodes.ToAPIErr(s3Err), r.URL)
+		return
+	}
+
+	userList := r.Form["users"]
```
Comment (shtripat) [inline cmd/admin-handlers-users.go]:
```suggestion
	users := r.Form["users"]
```

File: cmd/admin-handlers-users.go [hunk 4]
```diff
@@ -1165,6 +1165,173 @@ func (a adminAPIHandlers) DeleteServiceAccount(w http.ResponseWriter, r *http.Re
...
+			if err != nil {
+				writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
+				return
+			}
+			for _, sts := range stsKeys {
+				expiryTime := sts.Expiration
+				accessKeys.STSKeys = append(accessKeys.STSKeys, madmin.ServiceAccountInfo{
+					AccessKey:  sts.AccessKey,
+					Expiration: &expiryTime,
+				})
```
Comment (shtripat) [inline cmd/admin-handlers-users.go]:
```suggestion
				accessKeys.STSKeys = append(accessKeys.STSKeys, madmin.ServiceAccountInfo{
					AccessKey:  sts.AccessKey,
					Expiration: &sts.Expiration,
				})
```

File: cmd/admin-handlers-users.go [hunk 5]
```diff
@@ -1165,6 +1165,173 @@ func (a adminAPIHandlers) DeleteServiceAccount(w http.ResponseWriter, r *http.Re
...
+			if err != nil {
+				writeErrorResponseJSON(ctx, w, toAdminAPIErr(ctx, err), r.URL)
+				return
+			}
+			for _, svc := range serviceAccounts {
+				expiryTime := svc.Expiration
+				accessKeys.ServiceAccounts = append(accessKeys.ServiceAccounts, madmin.ServiceAccountInfo{
+					AccessKey:  svc.AccessKey,
+					Expiration: &expiryTime,
+				})
```
Comment (shtripat) [inline cmd/admin-handlers-users.go]:
```suggestion
				accessKeys.ServiceAccounts = append(accessKeys.ServiceAccounts, madmin.ServiceAccountInfo{
					AccessKey:  svc.AccessKey,
					Expiration: &svc.Expiration,
				})
```

File: cmd/admin-handlers-users.go (hunk 2 above)
Comment (taran-p) [reply cmd/admin-handlers-users.go:1246]: `users` is a map so slice expansion is not possible here

Comment (shtripat) [reply cmd/admin-handlers-users.go:1246]: oh ok

**Review by shtripat — APPROVED** [review]
overall looks good to me. tested with corresponding `mc` PR

**Review by harshavardhana — APPROVED** [review]

**Comment by harshavardhana** [comment]
Please resolve the conflicts @taran-p

**Comment by taran-p** [comment]
@harshavardhana resolved

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**